build-coordinator/src/main/java/org/jboss/pnc/core/builder/DatastoreAdapter.java

```diff
@@ -47,6 +46,10 @@
 
     private static final Logger log = Logger.getLogger(DatastoreAdapter.class);
 
+    // needed for EJB/CDI
```

Comment:
There are no EJBs available in BuildCoordinator, please update miss-leading comment.